documentation/modules/adding-source-provider.adoc

```diff
@@ -190,6 +190,14 @@ endif::[]
 +
 The provider appears in the list of providers.
 
+ifdef::vmware[]
++
+[NOTE]
+====
+It might take a few minutes for the provider to appear in the list.
```

Comment:
the provider should appear in the list immediately, it could take few minutes for it to reach a Ready state

Reply:
Fixed.